babel.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "corejs": 3,
+      "useBuiltIns": "entry",
+      "targets": {
+        "browsers": ["IE 11", "not dead"]
+      }
+    }],
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
```

Comment:
Consider adding some more plugins for completeness:
1. @babel/plugin-proposal-private-methods
2. @babel/plugin-proposal-decorators
3. @babel-plugin-proposal-optional-chaining

Reply:
I feel that is premature, given the general direction for this codebase. Let's hold off including this for now